flytekit/core/base_task.py

```diff
@@ -477,6 +479,8 @@ def __init__(
                 execution of the task. Supplied as a dictionary of key/value pairs
             disable_deck (bool): (deprecated) If true, this task will not output deck html file
             enable_deck (bool): If true, this task will output deck html file
+            decks (Tuple[str]): Tuple of decks to be
+                generated for this task. Valid values can be selected from fields of ``flytekit.deck.deck.DeckFields`` enum
```

Comment:
Can you reimport `DeckFields` in `flytekit.deck`?
```suggestion
                generated for this task. Valid values can be selected from fields of ``flytekit.deck.DeckFields`` enum
```